packages/timebar/src/timebar.js

```diff
@@ -5,6 +5,10 @@ import dayjs from 'dayjs'
 import memoize from 'memoize-one'
 import relativeTime from 'dayjs/plugin/relativeTime'
 import utc from 'dayjs/plugin/utc'
+import 'dayjs/locale/en'
```

Comment:
would be nice import them dynamically on locale change ? so we could avoid some bundle size but I've reviewed and all of them are 12kb so just a nice to have if it is very easy change